src/Xamarin.Android.Build.Tasks/Tasks/Aapt2.cs

```diff
@@ -34,7 +34,7 @@ public class Aapt2 : AsyncTask {
 
 		protected string ResourceDirectoryFullPath (string resourceDirectory)
 		{
-			return (Path.IsPathRooted (resourceDirectory) ? resourceDirectory : Path.Combine (WorkingDirectory, resourceDirectory)).TrimEnd ('\\');
```

Comment:
I still don't understand why this change is required.  The method name *implies* that a full path should be returned, which explains the original `Path.IsPathRooted()` and `Path.Combine(WorkingDirectory, ...)` calls, but now...  Presumably a relative path could be returned, but the method name still says "FullPath".

What's going on here?